source/index.md

```diff
@@ -597,11 +645,12 @@ otherwise, exactly one payment method must be used.
   <dd>[0-9]{2} <br /> Expiry month of card to charge.</dd>
   <dt>mobilepayonline[expire_year]</dt>
   <dd>[0-9]{4} <br /> Expiry year of card to charge.</dd>
-  <dt>mobilepayonline[phonenumber]</dt>
+  <dt>mobilepayonline[phone_number]</dt>
   <dd>
     [\x20-\x7E]{1,15} <br />
     <i>Optional when transaction is signed and partner is trusted.</i> <br />
-    Phone number from where the PAN originates.
+    Phone number from where the PAN originates. <br />
+    Randomly transactions will be selected and the translation will be verified.
```

Comment:
Suggestion: Random transactions will be selected for verification.

Reply:
I have already pushed this:
> Transactions will be randomly selected and the translation will be verified.

Could I ask you to refresh the page? 🙂

Reply:
Of couse. I was reviewing a specific commit (https://github.com/clearhaus/gateway-api-docs/pull/53/commits/933f5c3f109160e8ebe5096635ac62f38d813846).